lib/rubocop/cop/lint/eval.rb

```diff
@@ -8,9 +8,10 @@ class Eval < Cop
         MSG = 'The use of `eval` is a serious security risk.'
 
         def on_send(node)
-          receiver, method_name, = *node
+          receiver, method_name, *args = *node
 
-          add_offense(node, :selector) if receiver.nil? && method_name == :eval
+          return unless receiver.nil? && method_name == :eval && args.first.type != :str
```

Comment:
Btw, there's one more thing to consider - someone calling `eval` with no args. While it'd be a mistake, RuboCop would blow up here as `args.first` would return `nil`.